packages/orbit-components/playroom/snippets/BasicComponents.ts

```diff
@@ -54,6 +54,139 @@ const dialog = `
 
 const heading = `<Heading as="h1" type="title1">Heading</Heading>`;
 
+const linkList = `
+  <LinkList
+    direction="row"
+    spacing="400"
+    useMargin
+  >
+    <TextLink type="secondary">
+      Flight 1
+    </TextLink>
+    <TextLink type="secondary">
+      Flight 2
+    </TextLink>
+    <TextLink type="secondary">
+      Flight 3
+    </TextLink>
+  </LinkList>
+`;
+
+const list = `
+  <List>
+    <ListItem>
+      24,000 locations{' '}
+      <TextLink href="#">
+        around
+      </TextLink>
+      {' '}the globe
+    </ListItem>
+    <ListItem>
+      Lowest price car rental in{' '}
+      <strong>
+        Warsaw
+      </strong>
+    </ListItem>
+    <ListItem>
+      From 3 star budget to 5 star luxury
+    </ListItem>
+  </List>
+`;
+
+const listChoice = `
+<>
+  <ListChoice
+    description="Further description"
+    icon={<Icons.Accommodation />}
+    role="list"
+    selectable
+    tabIndex={0}
+    title="Choice Title"
+  />
+  <ListChoice
+    description="Further description"
+    icon={<Icons.Accommodation />}
+    role="list"
+    selectable
+    tabIndex={0}
+    title="Choice Title"
+  />
+</>
+`;
+
+const loading = `<Loading loading text="Please wait, content is loading..." type="pageLoader" />`;
+
+const navigationBar = `
```

Comment:
This should be on Layouts file, imo

Reply:
yeah, I was thinking about it as well. PR is just a `draft` for now.